wikitongues/wikitongues/data_store/airtable/airtable_language_data_store.py

```diff
@@ -93,7 +93,7 @@ def list_languages(self):
 
         result = ErrorResponse()
 
-        response = self._client.list_records()
+        response = self._client.list_records(page_size, max_records)
```

Comment:
This is calling [`IAirtableHttpClient.list_records`](https://github.com/wikitongues/Language-Indexing/blob/2901fe42829b7d0321dd854b496a7e7776396e64/wikitongues/wikitongues/data_store/airtable/airtable_http_client.py#L15) which has parameters `page_size`, `offset`, and `max_records`. Since we don't have `offset` here yet, we can call `list_records` using named arguments:
```
response = self._client.list_records(page_size=page_size, max_records=max_records)
```